src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -243,14 +245,7 @@ function BaseTextInput(
     // Some characters are wider than the others when rendered, e.g. '@' vs '#'. Chosen font-family and font-size
     // also have an impact on the width of the character, but as long as there's only one font-family and one font-size,
     // this method will produce reliable results.
-    const getCharacterPadding = (prefix: string): number => {
-        switch (prefix) {
-            case CONST.POLICY.ROOM_PREFIX:
-                return 10;
-            default:
-                throw new Error(`Prefix ${prefix} has no padding assigned.`);
-        }
-    };
+    const getCharacterPadding = (prefix: string): number => prefix.length * 10;
```

Comment:
Why do we have this changes?

Reply:
I'm guessing because we don't know the prefix width, it can be a single character or 3 depending on currency symbol, e.g. `$` for USD or `Dhs` for AED

Reply:
But with 3 prefix character, 30 will be huge padding for mobile devices 😱

Reply:
The padding doesn't actually look big, because the prefix takes its place as it's absolutely positioned in it. But we definitely need mobile screenshots to confirm